content/en/blog/_posts/2019-sig-docs-survey.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: blog
+title: "Kubernetes Documentation Survey"
+date: 2019-09-19
+slug: Kubernetes-Documentation-End-User-Survey
+---
+
+**Author:** Aimee Ukasick and SIG Docs
+
+In September, SIG Docs conducted its first survey about the [Kubernetes documentation](https://kubernetes.io/docs/). It was definitely a learning experience. We'd like to thank the CNCF's Kim McMahon for helping us create the survey and access the results.
+
+# Key takeaways
```

Comment:
Remove bullets from these points and make them separate paragraphs.

Reply:
ok... no "executive" bullet points?